Common/Source/Tracking/SkylinesGlue.cpp

```
@@ -8,16 +8,54 @@
+    uint64_t hex_to_uint64(const std::string& string) {
+        typedef std::is_same<uint64_t, unsigned long> is_long;
+        typedef std::is_same<uint64_t, unsigned long long> is_long_long;
+        static_assert(is_long::value || is_long_long::value, "invalid type");
+
+        try {
+            if constexpr (is_long::value) {
+                return std::stoul(string, 0, 16);
+            }
+            if constexpr (is_long_long::value) {
+                return std::stoull(string, 0, 16);
+            }
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)